french/exploratory/conv/outback_camp.rpy

```diff
@@ -4,407 +4,407 @@
 translate french outback_talk_jack_4a6660e2:
 
     # j nulla "Nice timing with that intervention, back there.{w=0.3} Now we have a chance."
-    j nulla ""
+    j nulla "Joli timing avec cette intervention, là-bas."
 
 # game/exploratory/conv/outback_camp.rpy:7
 translate french outback_talk_jack_4b55eec9:
 
     # d smile "It was my pleasure...{w=0.5} I had a feeling."
-    d smile ""
+    d smile "C'était un plaisir...{w=0.5} J'ai eu un bon préssentiment."
 
 # game/exploratory/conv/outback_camp.rpy:9
 translate french outback_talk_jack_df59f517:
 
     # j nulla "Yeah, she feels like a person who thinks highly of those who eventually push back..."
-    j nulla ""
+    j nulla "Ouais, elle se sent comme une personne qui a une haute opinion de ceux qui finissent par réagir..."
 
 # game/exploratory/conv/outback_camp.rpy:11
 translate french outback_talk_jack_e9c36ac6:
 
     # j nulla "Makes me wonder how the apprenticeship process went.{w=0.3} Think she gave Ruby a test, too?"
-    j nulla ""
+    j nulla "Ça me fait me demander comment le processus d'apprentissage est allé.{w=0.3} Tu penses qu'elle a fait passé un test à Ruby, aussi ?"
 
 # game/exploratory/conv/outback_camp.rpy:12
 translate french outback_talk_jack_382ff595:
 
     # d neutral "Wouldn't surprise me..."
-    d neutral ""
+    d neutral "Ça ne me surprendrai pas"
 
 # game/exploratory/conv/outback_camp.rpy:13
 translate french outback_talk_jack_33ace260:
 
     # d smile "But I think that Ruby just wore down her resistance with time."
-    d smile ""
+    d smile "Mais je pense que Ruby a juste baissé ça résistance avec le temps."
 
 # game/exploratory/conv/outback_camp.rpy:15
 translate french outback_talk_jack_5a4ecf97:
 
     # j nulla "Heh...{w=0.5} Sounds like her, alright.{w=0.3} I hope she's been well."
-    j nulla ""
+    j nulla "Eh...{w=0.5}Ça lui ressemble bien, d'accord.{w=0.3} J'espère qu'elle va bien."
 
 # game/exploratory/conv/outback_camp.rpy:24
 translate french outback_talk_jack_8dba4e6a:
 
     # j nulla "Yet another fight with a Gargantuan Grimm with way less people that I'm comfortable having..."
-    j nulla ""
+    j nulla "Encore un autre combat avec un Grimm Gargantuesque avec bien moins de monde avec qui je suis à l'aise..."
 
 # game/exploratory/conv/outback_camp.rpy:25
 translate french outback_talk_jack_650213ea:
 
     # d neutral "Maybe so...{w=0.5} But at least the quality has increased since Barito and Vodrunum."
-    d neutral ""
+    d neutral "Peut-être bien...{w=0.5} Mais au moins la qualité a augmenté depuis Barito et Vodrunum."
 
 # game/exploratory/conv/outback_camp.rpy:27
 translate french outback_talk_jack_ae2c5c38:
 
     # j nulla "I suppose...{w=0.5} Also, this time it's pretty straightforward search and destroy.{w=0.3} No one to rescue."
-    j nulla ""
+    j nulla "Je suppose...{w=0.5} De plus, cette fois c'est une recherche et destruction assez simple.{w=0.3} Personne à sauver."
```

Comment:
it's pretty straightforward search and destroy. -> c'est assez franchement "chercher et abattre".